packages/components/src/FormField/index.ts

```diff
@@ -1,6 +1,7 @@
 export * from "./FormField";
 
 export {
+  BaseFormFieldProps,
```

Comment:
do you think we can just `export *`

Reply:
Fixed here: https://github.com/GetJobber/atlantis/pull/627/commits/79b28dc4eb380af793cc5b25ac268904dc652b3c